src/components/homesections/homesections.js

```diff
@@ -334,12 +349,14 @@ define(['connectionManager', 'cardBuilder', 'appSettings', 'dom', 'apphost', 'la
 
     function getRequirePromise(deps) {
         return new Promise(function (resolve, reject) {
-            require(deps, resolve);
+            import(deps).then(() => {
+                return resolve;
+            });
         });
     }
 
-    function loadLibraryTiles(elem, apiClient, user, userSettings, shape, userViews, allSections) {
-        var html = '';
+    export function loadLibraryTiles(elem, apiCdflient, user, userSettings, shape, userViews, allSections) {
+        let html = '';
```

Comment:
apiCdflient?

Reply:
Welp that should be apiClient, thanks for noticing